package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "quick-pivot",
-  "version": "2.5.0",
+  "version": "2.5.1",
```

Comment:
Can you remove the version number change? I'll update this with `npm`.

Reply:
I discarded the change on the package.json file and force-pushed the updated commit.